index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Delicious: Chocolate Pizza</title>
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+</html>
```

Comment:
Style nitpick: GitHub wants you to leave a blank line at the end of each file, which is why there's this angry red symbol.